pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -581,6 +583,19 @@ func nodeAddressForLB(node *v1.Node) (string, error) {
 	return addrs[0].Address, nil
 }
 
+//getStringFromServiceAnnotation searches a given v1.Service for a specific annotationKey and either returns the annotation's value or a specified defaultSetting
+func getStringFromServiceAnnotation(service *v1.Service, annotationKey string, defaultSetting string) string {
+	glog.V(4).Infof("getStringFromServiceAnnotation(%v, %v, %v)", service, annotationKey, defaultSetting)
+	if annotationValue, ok := service.Annotations[annotationKey]; ok {
```

Comment:
@zetaab cool, then let's make it clear with comments that this is by design. So folks who come later don't try to "fix" things and break this scenarion. I'll `/approve` immediately after. thanks for your patience!